lib/pbench/test/unit/agent/task/test_results_move.py

```diff
@@ -363,23 +363,15 @@ def test_results_move_relay(monkeypatch, caplog, setup):
 
         # We expect two PUT calls using the relay base URI: a JSON manifest
         # object followed by the tarball. Responses stores "calls" in reverse
-        # order. While we can effect random access by searching for URIs, we
-        # can't easily know either SHA256 as the tarball and manifest are both
-        # generated internally. Instead, we traverse the list of calls in
-        # reversed order, expecting the first to resolve to a JSON file and the
-        # second to provide a URL corresponding to the tarball, which we assert
-        # must appear in the manifest "uri" field.
-        manifest = {}
-        uris = []
+        # order. We expect the first (index 1) to resolve to a JSON relay
+        # manifest file and the second (index 0) to provide a URL corresponding
+        # to the tarball, which we assert must appear in the manifest "uri"
+        # field.
```

Comment:
I'm not sure that Responses is using a counter-intuitive order, here:  doesn't the CUT actually upload the manifest last?

Reply:
Um... indeed. Server and Agent are swirling together in my brain and coming out sideways ... 😵‍💫 

Well, that's enough to push me over the edge to another commit. (Sigh.)